cosmos/Dockerfile

```diff
@@ -16,6 +18,7 @@ RUN mkdir -p lib/cosmos/ext \
   && apk add build-base ruby-dev libressl-dev \
   && bundle config set --local without 'development' \
   && bundle install --quiet \
+  && rake build \
   && rake gems \
   && gem install --local ./cosmosc2-*.gem \
```

Comment:
Gem gets installed right here.

Reply:
Yes but what version? The problem I was running into was I couldn't run with my new code. Also the CI/CD pipeline failed because it wasn't using the new code either.

Reply:
Ok.  The issue is the api Gemfiles aren't using prelease versioned gems.   I will fix.